tests/components/august/test_init.py

```diff
@@ -127,8 +151,7 @@ def _lock_return_activities_side_effect(access_token, device_id):
         "homeassistant.components.august.config_flow.AugustGateway.async_authenticate",
         return_value=True,
     ):
-        mocked_config = _mock_get_config()
-        assert await async_setup_component(hass, "august", mocked_config)
+        assert await async_setup_component(hass, DOMAIN, _mock_get_config())
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5156575)